sdk/auth/client.go

```diff
@@ -0,0 +1,14 @@
+package auth
+
+import (
+	"net/http"
+
+	"github.com/hashicorp/go-retryablehttp"
+)
+
+// httpClient returns a shimmed retryablehttp Client, at the moment with default retry,
+// backoff and timeout settings, but we may need to change this in the future.
+func httpClient() *http.Client {
+	retryableClient := retryablehttp.NewClient()
```

Comment:
double-checking this includes the ProxyFromEnvironment setting?

Reply:
Fortunately it does, thanks to very [usable defaults](https://github.com/hashicorp/go-cleanhttp/blob/02f12f05b908335f65f124c1a7f1ec45f5c42a35/cleanhttp.go#L26-L42) in go-cleanhttp.